src/client/View.ts

```diff
@@ -0,0 +1,147 @@
+// @ts-ignore
+import { Stage } from "konva/lib/Stage";
```

Comment:
Does konva not have typescript types?

Reply:
Yes. Konva have type declaration files, but it's not supported by default. https://github.com/konvajs/konva/issues/749